internal/store/db/redis/store.go

```diff
@@ -200,7 +201,7 @@ func (c Client) Disconnect() error {
 }
 
 // NewClient provides a factory for building a StoreClient
-func NewClient(config db.DatabaseInfo) (interfaces.StoreClient, error) {
+func NewClient(config db.DatabaseInfo, credentials bootstrapConfig.Credentials) (interfaces.StoreClient, error) {
```

Comment:
I would prefer the credentials absorbed into db.DatabaseInfo in terms of NewClient constructor point of view so that the StoreClient NewClient only take dbInfo.  Any reason to separate this into two objects?

Reply:
They where separated out because they are no longer configuration.  db.DatabaseInfo is configuration in the TOML file. Credentials are secrets in the Secret Store.

Reply:
I see, that sounds good to me then.